utils/common/src/main/java/org/apache/brooklyn/util/yaml/Yamls.java

```diff
@@ -151,14 +151,14 @@ public static void dump(int depth, Object r) {
 
     /** simplifies new Yaml().loadAll, and converts to list to prevent single-use iterable bug in yaml */
     @SuppressWarnings("unchecked")
-    public static Iterable<Object> parseAll(String yaml) {
+    public static List<Object> parseAll(String yaml) {
```

Comment:
prefer the return type being `Iterable` as that allows lazy eval.  callers who need a list can do `MutableList.copyOf(Iterable)`.